From MAILER-DAEMON  Wed Oct 27 18:15:02 2010
X-Original-To: pragone@pragone.no-ip.org<pragone@pragone.no-ip.org>
Date: Wed, 27 Oct 2010 18:15:02 +0200 (CEST)
Delivered-To: foo@asdf.com
From: MAILER-DAEMON@pragone.no-ip.org<MAILER-DAEMON@pragone.no-ip.org> (Mail Delivery System)
Subject: Undelivered Mail Returned to Sender
To: pragone@pragone.no-ip.org<pragone@pragone.no-ip.org>
Auto-Submitted: auto-replied
MIME-Version: 1.0
Content-Type: multipart/report; report-type=delivery-status;
        boundary="5B5A25C08DB.1288196102/pragone.no-ip.org"

This is a MIME-encapsulated message.

--5B5A25C08DB.1288196102/pragone.no-ip.org
Content-Description: Notification
Content-Type: text/plain; charset=us-ascii

This is the mail system at host pragone.no-ip.org<http://pragone.no-ip.org>.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<cvbn34c4c097@gmail.com<cvbn34c4c097@gmail.com>>: host gmail-smtp-in.l.google.com[209.85.227.27] said:
    550-5.1.1 The email account that you tried to reach does not exist. Please
    try 550-5.1.1 double-checking the recipient's email address for typos or
    550-5.1.1 unnecessary spaces. Learn more at
    550 5.1.1 http://mail.google.com/support/bin/answer.py?answer=6596
    eq4si15922233wbb.79 (in reply to RCPT TO command)

--5B5A25C08DB.1288196102/pragone.no-ip.org
Content-Description: Delivery report
Content-Type: message/delivery-status

Reporting-MTA: dns; pragone.no-ip.org<http://pragone.no-ip.org>
X-Postfix-Queue-ID: 5B5A25C08DB
X-Postfix-Sender: rfc822; pragone@pragone.no-ip.org<pragone@pragone.no-ip.org>
Arrival-Date: Wed, 27 Oct 2010 18:14:58 +0200 (CEST)

Final-Recipient: rfc822; cvbn34c4c097@gmail.com<cvbn34c4c097@gmail.com>
Action: failed
Status: 5.1.1
Remote-MTA: dns; gmail-smtp-in.l.google.com<http://gmail-smtp-in.l.google.com>
Diagnostic-Code: smtp; 550-5.1.1 The email account that you tried to reach does
    not exist. Please try 550-5.1.1 double-checking the recipient's email
    address for typos or 550-5.1.1 unnecessary spaces. Learn more at
    550 5.1.1 http://mail.google.com/support/bin/answer.py?answer=6596
    eq4si15922233wbb.79

--5B5A25C08DB.1288196102/pragone.no-ip.org
Content-Description: Undelivered Message
Content-Type: message/rfc822

Return-Path: <pragone@pragone.no-ip.org<pragone@pragone.no-ip.org>>
Received: by pragone.no-ip.org<http://pragone.no-ip.org> (Postfix, from userid 1000)
        id 5B5A25C08DB; Wed, 27 Oct 2010 18:14:58 +0200 (CEST)
To: cvbn34c4c097@gmail.com<cvbn34c4c097@gmail.com>
Subject: Test
Message-Id: <20101027161458.5B5A25C08DB@pragone.no-ip.org<20101027161458.5B5A25C08DB@pragone.no-ip.org>>
Date: Wed, 27 Oct 2010 18:14:58 +0200 (CEST)
From: pragone@pragone.no-ip.org<pragone@pragone.no-ip.org> (Paolo Ragone)


test

--5B5A25C08DB.1288196102/pragone.no-ip.org--
